src/lando/main/admin.py

```diff
@@ -2,6 +2,10 @@
 from typing import Callable, Self
 
 from django.contrib import admin
+from django.db.models import Field as DbField
+from django.db.models import Field as FormField
```

Comment:
Both of these imports are aliasing `django.db.models.Field`, seems the latter should be aliasing `django.forms.Field` instead.